commands/command.go

```diff
@@ -30,20 +30,21 @@ var Verbs = map[string]func(api.API) Command{
 	"paste":     NewPaste,
 	"pause":     NewPause,
 	"play":      NewPlay,
-	"prev":      NewPrevious,
 	"previous":  NewPrevious,
+	"prev":      NewPrevious,
```

Comment:
I'd suggest leaving this list sorted. AFAIK "prev" should be sorted before "previous"?